tests/Unit/Evolution/Systems/CurvedScalarWave/Worldtube/ElementActions/Test_Iterations.cpp

```diff
@@ -130,7 +130,7 @@ void test_iterations(const size_t max_iterations) {
   MAKE_GENERATOR(generator);
   std::uniform_real_distribution<> dist(-10., 10.);
   std::uniform_real_distribution<> pos_dist(2., 10.);
-  std::uniform_real_distribution<> vel_dist(-0.3, 0.3);
+  std::uniform_real_distribution<> vel_dist(-0.2, 0.2);
```

Comment:
Could you add a note about why the magnitude of the velocity can't be larger than 0.2? I'm guessing too large a velocity causes something (the worldtube?) to move too much

Reply:
I am actually not sure why, as the worldtube is still going slower than the speed of light. I have not debugged what breaks then in this test but this was never encountered in a simulation.